### DISCUSSION TITLE
Brackets 2.0.1 Release

### DISCUSSION THREAD

**Discussion by abose** [brackets-cont/brackets#90] [Announcements]
**_Brackets 2.0 Release is here!_**
* Download pre-release Windows installer: [brackets.2.0.0.prerelease.exe](https://github.com/brackets-cont/brackets/releases/download/v2.0.0-prerelease/brackets.2.0.0.prerelease.exe)
* Mac installer: [TBD soon]()

##  This pre-release includes
1. New installer experience for windows.
2. New brackets extension store maintained by the community.
3. Performance improvement for the files panel.
4. Support large projects with more than 30,000 files.
5. Improved extension compatibility.

<hr /><em>This discussion was created from the release <a href='https://github.com/brackets-cont/brackets/releases/tag/'>Brackets 2.0.1 Release</a>.</em>

**Comment by arlindocodinha** [comment]
Hy Guys.
I tested version 2.01 on another Mac latest OS version mac .

![image](https://user-images.githubusercontent.com/27304872/144610743-7d3dbeb2-6e0e-4f60-93e1-abccaaed3822.png)

![image](https://user-images.githubusercontent.com/27304872/144610830-9e91a6a4-91be-4f38-9d86-5d77511d280d.png)

**Comment by arlindocodinha** [comment]
And you have to force quit, will not close the application using normal procedures. This issue comes from old versions since 1.12 or 1.13 I do not remember.

**Reply by charlypa** [reply]
@arlindocodinha 
Apologies for the issue. Could you verify if this issue exists in an older version of the brackets
https://github.com/adobe/brackets/releases/tag/release-1.14.2

**Comment by KLukan** [comment]
Greetings. I'm getting the same blank gray screen as demonstrated by @arlindocodinha.
<img width="576" alt="Screen Shot 2021-12-03 at 1 06 25 PM" src="https://user-images.githubusercontent.com/94011358/144658952-8691dbdf-fbcc-45e4-813e-677660707f55.png">

**Reply by charlypa** [reply]
@KLukan  Thanks for reporting the issue. We have raised a point to track the same
https://github.com/brackets-cont/brackets/issues/94

**Reply by charlypa** [reply]
Apologies for the issue. Could you verify if this issue exists in an older version of the brackets
https://github.com/adobe/brackets/releases/tag/release-1.14.2

**Reply by charlypa** [reply]
@KLukan 
 We have fixed the issue. Could you download the latest version and try it out?

**Reply by KLukan** [reply]
@charlypa 
So I loaded up the latest version, and it does start up...however I get this error screen. Please note, that I was getting the same error in my previous 1.x version...not sure what it is.

<img width="622" alt="Screen Shot 2021-12-05 at 12 22 57 PM" src="https://user-images.githubusercontent.com/94011358/144758713-e2636b35-c2e3-4b61-9f66-4ecc2c4fd6d1.png">

**Reply by charlypa** [reply]
@KLukan 
Apologies for the issue.
Have you had PHP installed on your machine?

**Reply by KLukan** [reply]
@charlypa
Yes. PHP 7.3 is installed

**Reply by charlypa** [reply]
@KLukan 
Have you had any PHP extensions installed in brackets?

**Reply by KLukan** [reply]
@charlypa 

If you mean Bracket plugins...yes perhaps I did some time ago in a v1 version. But this v2 install was clean...didn't add anything to it.

**Reply by charlypa** [reply]
@KLukan 
Created an issue to track the same
https://github.com/brackets-cont/brackets/issues/95

**Comment by feylen** [comment]
## Non-Functional Application
I thought it was just my M1 Mac doing weird things, but then I saw another M1 on this thread having the problem. Same with Monterey, but dido. So - here is my grey window. On the bright side - the grey window does open very fast!!

<img width="1440" alt="Screen Shot 2021-12-04 at 12 58 44 PM" src="https://user-images.githubusercontent.com/79057699/144721394-c194305a-7d8f-4be4-86a8-cb71eda6c529.png">

**Reply by charlypa** [reply]
Apologies for the issue. Could you verify if this issue exists in an older version of the brackets
https://github.com/adobe/brackets/releases/tag/release-1.14.2

**Reply by charlypa** [reply]
@feylen  we have fixed the issue. Could you download the latest version and try it out

**Comment by arlindocodinha** [comment]
Tto me still the same grey window. Is it the build 2.0.1? version 1.14.2 exusts the same issue and even before. This issue started 1.13 or even before  but I'm not sure.

**Reply by charlypa** [reply]
@arlindocodinha 
Apologies for the issue.
To debug the problem. Would you please do the following steps?

1. Uninstall all exiting brackets from your machine
2. install the pre-release version of brackets https://github.com/brackets-cont/brackets/releases/download/v2.0.0-prerelease/Brackets.Release.2.0.0_rc.dmg
3. Open Terminal and  run the following command   ```xattr -cr /Applications/Brackets.app```
4. Launch the brackets
5. See if brackets launch successfully or not

**Reply by arlindocodinha** [reply]
Now is working without any problem. works with the another builds too?
Thank you so much

**Reply by arlindocodinha** [reply]
But after a few minutes same problem and doesn't open again

**Reply by charlypa** [reply]
@arlindocodinha 
Applogies for the inconvience.
Mac installer had an issue with signature. Now we fixed it.
Would you please do the following steps to resolve the problem?

1. Uninstall  brackets 2.0.1
2. Download 2.0.1 again from [bracket.10](https://brackets.io/)
3. Install the newly downloaded version
4. See if it works or not

**Comment by arlindocodinha** [comment]
brackets 2.0.1 signed.dmg doesn’t work grey screen.
thank you

**Comment by Syndicoon** [comment]
Hi Everyone, I had to reload my linux system recently and now I can't get brackets. I see windows and mac installers but only source code for linux. Could a .deb or .rpm package be built? If not that, some directions on how to compile the code would be helpful. Thanks!

**Reply by abose** [reply]
Hi @Syndicoon Sorry for the inconvenience. 
Please use the installers available in https://github.com/adobe/brackets/releases/tag/release-1.14.1
That linux version does not support extension store and you might have to resolve unmet deps.

We are doing a larger platform migration for enhanced Linux support here: https://github.com/brackets-cont/brackets/projects/3
That is however a few months away at this time.

**Reply by arjobsen** [reply]
Unfortunately it depends on libcurl3 which is not maintained by my distro (Pop OS) anymore. Only libcurl4 is installable through apt. Thats too bad

**Comment by zash13x** [comment]
![image](https://user-images.githubusercontent.com/7418104/147337251-0f97c505-32b1-4ccc-b180-692935a67f7e.png)
It takes 10 years to install.
- not a fresh install
- updating from the latest version of the old brackets (the one not using the new installer)

**Reply by abose** [reply]
Hopefully we will be able to improve it once we port to electron.

**Comment by tikendraw** [comment]


**Reply by charlypa** [reply]
@tikendraw 
Currently, brackets is not natively supported in Linux. You could use wine to run it on Linux. 
We are working on a major platform upgrade once its done Linux will be supported.  Please refer to this issue to track the same
https://github.com/brackets-cont/brackets/issues/112

**Comment by mohammed199708** [comment]
Hi
can I compile this brackets version on Linux using this tutorial https://github.com/adobe/brackets-shell/wiki/Building-brackets-shell#linux
if not how I can use it without Wine?
thanks ^__^

**Reply by charlypa** [reply]
@mohammed199708 
1. You can install wine from  https://wiki.winehq.org
2. Download windows installer for brackets https://github.com/brackets-cont/brackets/releases/download/v2.0.1/Brackets.2.0.1ml.exe
3. Open terminal type `wine ./Brackets.2.0.1ml.exe`
4. Open brackets and start using it

**Comment by Abedstark** [comment]
there is an error from my end it says install PHP 7 runtime please help me Mac M1 monterey
<img width="1030" alt="Screenshot 2022-03-05 at 10 38 32 PM" src="https://user-images.githubusercontent.com/98378884/156892168-53c4fad9-dbee-4724-a569-ed836b302cce.png">

**Reply by mlaungani** [reply]
HI guys! 
any update on this? 
Facing the same issue. 

The solution for the same on  a windows system is available, but not able to fix it on Mac os. 

TIA for a solution.

**Comment by cnrkuo** [comment]
Hmm... Any Debian build there?
